### DISCUSSION TITLE
QOL for managing collections

### DISCUSSION THREAD

**Discussion by henson-hugh** [ppy/osu#30708] [Ideas] [answered]
I noticed it's a bit hard to find the manage collections component in lazer. A player gets to 'Manage Collections' by scrolling to the bottom of the header dropdown or right click on a map then scroll to the bottom of the collections. 

Could 'Manage Collections' be placed in the same level as the first dropdown with:
Play, Edit, Details, Collections, {Manage Collections}, Copy Link, Hide
Or maybe have 'Manage Collections' at the top of the collections list in the expanded view of 'Collections >'?

![osu_2024-11-16_18-34-55](https://github.com/user-attachments/assets/0e1f0224-d875-4a52-8431-2776a3132aa2)

![osu_2024-11-16_18-41-13](https://github.com/user-attachments/assets/15ff8796-4e6a-4736-b252-e2547f77b798)

Comparing this to stable that I was used to where the 'Manage Collections' option shows at the top of the screen when you right click a map, lazer has one more click plus some scrolling.
![screenshot014](https://github.com/user-attachments/assets/5909d870-e60c-4330-ac42-74edb3530a1b)

Also, to create a new collection you have to scroll to the bottom of the collections. 
![osu_2024-11-16_18-33-51](https://github.com/user-attachments/assets/b3199ed7-ae1c-4611-847b-e80b88d512f0)

Stable has the input box at the top and the player doesn't have to scroll at all.
![screenshot015](https://github.com/user-attachments/assets/494925ca-20fb-4f2b-840f-e911377a1bba)

The 'Create a new collection' option could be moved to top for ease of access.

**Comment by peppy** [comment]
You can also access this dialog with two clicks:


https://github.com/user-attachments/assets/4bf4fd55-fafd-4887-941d-4c2212d93e37


Or alternatively, `F3` -> `1` (same as stable).

**Reply by peppy** [reply, marked as answer]
Other concerns should be addressed by https://github.com/ppy/osu/pull/30743.
